addons/sourcemod/scripting/hosties/lastrequest.sp

```diff
@@ -551,6 +553,8 @@ LastRequest_OnPluginStart()
 	gShadow_LR_BlockSuicide = false;
 	gH_Cvar_LR_VictorPoints = CreateConVar("sm_hosties_lr_victorpoints", "1", "Amount of frags to reward victor in an LR where other player automatically dies", 0, true, 0.0);
 	gShadow_LR_VictorPoints = 1;
+	gH_Cvar_LR_RemoveArmor = CreateConVar("sm_hosties_lr_removearmovr", "1", "Remove armor before LR", 1, true, 0.0);

```

Comment:
Why is this a 1 and all others are 0 for the convar flags?  Originally these were all FCVAR_PLUGIN.  I'm not sure why we changed to magic numbers.